docs/docs/documentation/getting-started/installation/sqlite.md

```diff
@@ -10,11 +10,11 @@ SQLite is a popular, open source, self-contained, zero-configuration database th
 
 ```yaml
 ---
-version: "3.7"
```

Comment:
Why was the version removed?

Reply:
The version property has been deprecated for some time, and newer versions of compose now output a warning. Compose v1 reached EOL status years ago.

Reply:
Yeesh. Good to know thanks!